src/Frontend/Page/Topic.hs

```diff
@@ -23,11 +23,12 @@ where
 import Action (ActionM, ActionPersist(..), ActionUserHandler, ActionExcept, renderContext)
 import Control.Exception (assert)
 import Frontend.Page.Category
+import Frontend.Page.Overview
```

Comment:
One page module should not include other one.We have an issue #346 connected to this change. But for the time being it is ok.